CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# 0.11.0
+
+## New Features
+
+- Creating of Pagure issues now supports tags. (@cverna)
+- Project issues can now be filtered by labels. (@cverna)
+- GitProject has new is_private() method. (@dhodovsk)
+- Tokens & keys are now obfuscated in logs. (@lachmanfrantisek)
+- PR classes now have diff_url property. (@lachmanfrantisek)
```

Comment:
I only rebase [PR by pawelkopka](https://github.com/packit-service/ogr/pull/280) -- credits should go to him...